tests/Cucumber/extra_testdata/good/feature_keyword_in_scenario_description.feature

```diff
@@ -0,0 +1,4 @@
+Feature: One
+  Scenario: Something
+
+Feature: Two
```

Comment:
Do we also have a test (maybe upstream?) covering a file that would try to put 2 _meaningful_ features ? In that case, the `Feature` keyword would not be part of the description of something (as steps don't have description)

Reply:
@stof does this one cover it (also added in this PR, but as a "bad" example)? https://github.com/Behat/Gherkin/pull/323/files#diff-82e1f68b1699d4413bab996af3f48d6d66c34e1cece5802ddc5ce31c8e931ad5

It's a special kind of input, but it doesn't require special handling. After the first `Feature:` tag then a line that begins `Feature:` is just text content, and therefore valid / invalid depending on whether the current position in the document allows free text. This appears to be what cucumber/gherkin is doing.

Reply:
yes, this covers it and matches what I thought.

The existing code of the Parser indeed handles it (thanks to its handling of unexpected token at that position), but I think it is good to actually cover it with a "bad" example.